src/main/java/nl/tudelft/contextproject/util/EntityUtil.java

```diff
@@ -20,9 +20,9 @@ private EntityUtil() {}
      * Get the ID for a certain class.
      *
      * @param entity
-     *          the classname of the entity
+     *      the classname of the entity

```

Comment:
File has spaces indentation instead of tabs. Really strange that we have so many files that have this. Eclipse and IntelliJ should both default to tabs only.